harness/determined/cli/job.py

```diff
@@ -1,56 +1,75 @@
 import json
 from argparse import Namespace
+from datetime import datetime
 from typing import Any, List
 
+import pytz
 import yaml
 
 from determined.cli import render
 from determined.cli.session import setup_session
+from determined.common import api
 from determined.common.api import authentication
 from determined.common.api.b import get_GetJobs
 from determined.common.declarative_argparse import Arg, Cmd, Group
 
 
 @authentication.required
 def ls(args: Namespace) -> None:
+    priority = True
+    config = api.get(args.master, "config").json()
+    try:
```

Comment:
suggestion: it seems safer and perhaps easier to me to go through the resource pool endpoint (the cli wouldn't need to assume this nested structure schema for the config.)